docs/supported_apis/dataframe_supported.rst

```diff
@@ -328,7 +328,7 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``select_dtypes``          | `select_dtypes`_          | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``sem``                    | `sem`_                    | D                      |                                                    |
+| ``sem``                    | `sem`_                    | Y                      |                                                    |
```

Comment:
How has performance increased on the local machine?

Reply:
Here are some run times of `sem` with usage of a machine which I work on:
shape             |   time of pandas_df.sem   |   time of modin_df.sem()   |
----------------------------------------------------------------------------
2^18 x 2 ^ 6  |   0.7121448516845703     |   0.6343734264373779      |
----------------------------------------------------------------------------
2^19 x 2 ^ 6  |   1.7079746723175049     |   1.255998134613037        |
----------------------------------------------------------------------------
2^20 x 2 ^ 6  |   3.0300698280334473     |    2.4800331592559814     |

Reply:
good! Thanks @YarShev